src/PresentationalComponents/AppIntro/AppIntro.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  Button,
+  Label,
+  Popover,
+  PopoverPosition,
+} from '@patternfly/react-core';
+import IntroBody from './IntroBody';
+import messages from '../../Messages';
+import { useIntl } from 'react-intl';
+
+const AppIntro = () => {
+  const intl = useIntl();
+  const [show, setShow] = React.useState(false);
+
+  const shouldClose = () => {
+    setShow(false);
+  };
+  const shouldOpen = () => {
+    setShow(true);
+  };
+
+  const HeaderContent = () => (
+    <div>{intl.formatMessage(messages.introExploreInsights)}</div>
+  );
+
+  return (
+    <Popover
+      aria-label="Advisor intro wizard"
+      position={PopoverPosition.bottomEnd}
+      hideOnOutsideClick={false}
+      isVisible={show}
+      shouldClose={shouldClose}
```

Comment:
You can remove `shouldClose` and `shouldOpen` functions in this component and simply pass `setShow` to the `shouldClose` and `shouldOpen` properties like so:
```
shouldClose={() => setShow(false)}
shouldOpen={() => setShow(true)}
```